scripts/install-deps.sh

```diff
@@ -527,7 +527,7 @@ do_install_azure_storage_sdk() {
 
     cmake "${azure_storage_sdk_cmake_options[@]}" . || return
 
-    cmake --build . || return
+    cmake --build . || return 1
```

Comment:
Is it better to overwrite (with 1) the cmake exit code on failure? line 531 for instance does not. Which commit in main is this from?

Reply:
Fixed